Main.py

```diff
@@ -51,18 +54,21 @@ def App():
                     articleNumber=raw_input("Article No >>>>")
                     articleNumber=int(articleNumber)
 
+                    if articleNumber==99 :
+                        sys.exit()
+                    if articleNumber==66 :

```

Comment:
It will not work as expected.
If the user enters 66 then it will ask a user to select another article but the intention of the user was to go to the main menu to select different news source.
Small fix needed  something on the lines of 
if article no is 66 or 99 break from the immediate loop and then decide whether to exit or go back to the main menu.